docs/guides/in-product-word-list.mdx

```diff
@@ -0,0 +1,534 @@
+controlling the availability of it on an account.
+
+| **✅ Do **              | **❌ Don't**           |
+| ----------------------- | ---------------------- |
+| Turn on Jobber Payments | Enable Jobber Payments |
+
+When referencing the apps in the App Marketplace, use **connect** and
+**disconnect** for controlling their availability.
+
+| **✅ Do **        | **❌ Don't**      |
+| ----------------- | ----------------- |
+| Connect Mailchimp | Turn on Mailchimp |
+
+<br />
+
+### export / import / download / upload
+
+Use **export** and **import** when data is moving in or out of Jobber and is
+converted into a different format.
+
+| **✅ Do **    | **❌ Don't**    |
+| ------------- | --------------- |
+| Import CSV    | Upload CSV      |
+| Export Report | Download Report |
+
+Use **download** and **upload** when data is being transferred in or out of
+Jobber in the same format.
+
+| **✅ Do **           | **❌ Don't**       |
+| -------------------- | ------------------ |
+| Upload photo         | Import photo       |
+| Download attachments | Export attachments |
+
+<br />
+
+### got it
+
+Use **got it** as a call to action when a user can acknowledge a message without
+taking action on it, like in promo cards or onboarding.
+
+<br />
+
+### link / attach / copy / clone / duplicate
+
+Use **link** when the same information is shown elsewhere in Jobber but not
+copied to create a new version.
+
+| **✅ Do**                          | **❌ Don't**                       |
+| ---------------------------------- | ---------------------------------- |
+| Link quote note to related invoice | Copy quote note to related invoice |
+
+Use **copy** when Jobber creates a new version that would have no effect on the
+original when edited.
+
+| **✅ Do**  | **❌ Don't**    |
+| ---------- | --------------- |
+| Copy Quote | Duplicate Quote |
+
+Use **attach** when describing receipts, forms or signatures being added to an
+object. For file selection, use **select a file** for consistency.
+
+| **✅ Do**                                                         | **❌ Don't**                                                      |
+| ----------------------------------------------------------------- | ----------------------------------------------------------------- |
+| Attach a receipt to this expense <br /> [Select a file](#actions) | Attach a receipt to this expense <br /> [Attach a file](#actions) |
+
+<br />
+
+### save / update / submit
+
+Use **save** at the end of a form on completion or **save changes** on editing.
+
+| **✅ Do**               | **❌ Don't** |
+| ----------------------- | ------------ |
+| Save Job / Save Changes | Update Job   |
+
+Use **update** when individual changes or groups of changes can be saved inline
+without needing to save a whole view or page.
+
+| **✅ Do**            | **❌ Don't**       |
+| -------------------- | ------------------ |
+| Update Company Hours | Save Company Hours |
```

Comment:
Is this in reference to Business hours in company settings? If so does it make sense to highlight an example that is showing that we're currently doing it incorrectly in product?

Reply:
Yeah, most of my "don't"s are from currently in product 😅

Reply:
I am going to change this to "Business" though as it isn't "Company" thats the wrong part here which can be misleading